lib/rules/expect-single-argument.js

```diff
@@ -14,7 +14,7 @@ function create (context) {
             message: 'Expect must have a single argument. More than one argument were provided.',
             node
           })
-        } else if (node.arguments.length === 0 && node.parent.property.name !== 'nothing') {
+        } else if (node.arguments.length === 0 && node.parent?.property?.name !== 'nothing') {
```

Comment:
I think the parent-? is not needed, as the CallExpression is always child of something, at least an ExpressionStatement. But it also does not hurt. The property-? fixes it.

Reply:
It is indeed unnecessary. I've removed the optional chaining operator after the parent.